core/selection.js

```diff
@@ -2495,7 +2495,7 @@
 		 */
 		scrollIntoView: function() {
 			// Scrolls the first range into view.
-			if ( this.type != CKEDITOR.SELECTION_NONE )
+			if ( this.getType() != CKEDITOR.SELECTION_NONE )
 				this.getRanges()[ 0 ].scrollIntoView();
```

Comment:
Please, [always use brackets for a single line statement](https://github.com/ckeditor/ckeditor4/blob/master/dev/docs/codestyle.md#2212-always-use-brackets-even-for-a-single-line-statements).